web/client/codechecker_client/cmd_line_client.py

```diff
@@ -1037,9 +1039,8 @@ def get_report_show_info(report):
             checked_file = file_name \
                 + ':' + str(bug_line) + ":" + str(bug_col)
             check_msg = report.main['description']
-            source_line =\
-                get_line_from_file(report.main['location']['file_name'],
-                                   bug_line)
+            if os.path.exists(file_name):
+                source_line = get_line_from_file(file_name, bug_line)
```

Comment:
if the file exists (and changed) the line can be read from it if I'm right, probably the wrong line will be read from the file and it will not be show as a changed file (because some text was read from the file)?

Reply:
in the util package there is another function to get a specific line from a file